elastic/elastic-agent
# integration:local can run a single test

## What does this PR do?
The `mage integration:local` command now supports as an optional
argument the name of a test to run. This test name will be passed to
`go test` as `--run=<test name>`. If no argument is passed, `mage


## Why is it important?

It enhances the developer experience.

## Checklist

~~- [ ] I have made corresponding change to the default configuration files~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have added an integration test or an E2E test~~

~~## Author's Checklist~~
## How to test this PR locally
Run `mage integration:local <test name>`, only the selected, local, test will run

~~## Related issues~~
~~## Use cases~~
~~## Screenshots~~
~~## Logs~~
~~## Questions to ask yourself~~
~~- How are we going to support this in production?~~
~~- How are we going to measure its adoption?~~
~~- How are we going to debug this?~~
~~- What are the metrics I should take care of?~~
